tap_facebook/streams.py

```diff
@@ -98,7 +98,7 @@ class AdsInsightStream(FacebookStream):
     path = f"/insights?level=ad&fields={columns}"
 
     replication_keys = ["date_start"]  # noqa: RUF012
-    replication_method = "incremental"
+    replication_method = "INCREMENTAL"
```

Comment:
We could probably make this even safer by using `singer_sdk.streams.core.REPLICATION_INCREMENTAL`